README.md

```diff
@@ -24,18 +24,20 @@ CSIP depends on the [SCIP Optimization Suite](http://scip.zib.de/#scipoptsuite).
 Starting with release 0.4.0, **CSIP only supports SCIP Optimization Suite
 4.0.0** or newer.
 
-[Download](http://scip.zib.de/download.php?fname=scipoptsuite-4.0.0.tgz)
-and extract the source files, then build the shared library
-(containing SCIP and SoPlex) with
-
-    make SHARED=true GMP=false READLINE=false ZLIB=false scipoptlib
+[Download](http://scip.zib.de/download.php?fname=scipoptsuite-5.0.0.tgz)
+the SCIP Optimization Suite and extract the source files.
+Build the shared library (containing SCIP and SoPlex) with `cmake` via
+
+    mkdir build
+    cd build
+    cmake ..
+    make
     
-which will produce a file `libscipopt.so`.
 
 To build CSIP, set the environment variable `SCIPOPTDIR` to point to the
-directory that contains the `scipoptsuite` sources. CSIP needs the library in
-`${SCIPOPTDIR}/lib/scipoptlib.so` and the C header files in
-`${SCIPOPTDIR}/scip-*/src/`.
+directory used to build `scipoptsuite`, that is, `<path to SCIP Optimization Suite>/build`.
```

Comment:
I don't think the build dir is relevant. If users do a `make install` then `SCIPOPTDIR` should point to `/usr` or something.

Reply:
So, I just tried the `DESTDIR=xyz make install` of the `cmake` way. Instead of installing everything to `xyz/`, it actually went to `xyz/usr/local/`. Is that a bug?

Also, when I used `xyz/usr/local` as `SCIPOPTDIR`, the `make` of CSIP did not work, because there is the extra `include` dir in the path to the headers.

I don't know what to make of it. Should we just use the old Makefiles instead?

Reply:
I looked at the output of `cmake` again. In the `build` folder, there is a `scip/src` folder, but it does not contain the header files. I guess that means that using a single env variable `SCIPOPTDIR` does not work anymore, we have to specify one for the headers and one for the library?